notebooks/00_quick_start/sar_single_node_movielens.ipynb

```diff
@@ -41,7 +41,7 @@
   },
```

Comment:
<p>This looks better after the refactoring  of the indexing. However, I think it is good to keep the logging of the progress of the algorithm.</p>

 _Reply via <a href='https://app.reviewnb.com/Microsoft/Recommenders/pull/485/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>I'm not sure I understand, do you mean surfacing the info level logs from sar_single_node in the notebook? Or just delineating how much time is taking for creating the indices vs the other portion of the algorithm?</p>

Reply:
Not the time, just the info about what part of the algorithm is under progress i.e.
"Collecting user affinity matrix... 
Calculating time-decayed affinities... 
Creating index columns... 
Building user affinity sparse matrix... 
Calculating item cooccurrence... 
Calculating item similarity... 
Calculating jaccard... 
Calculating recommendation scores... 
done training"

Reply:
ok, np